src/pytest/test.py

```diff
@@ -8,26 +8,27 @@
 from RLTest import Env
 from includes import *
 
-def testAdd(env):
-    if env.is_cluster():
-        raise unittest.SkipTest()
-
-    r = env
-    env.assertOk(r.execute_command(
-        'ft.create', 'idx', 'schema', 'title', 'text', 'body', 'text'))
-    env.assertTrue(r.exists('idx:idx'))
-    env.assertOk(r.execute_command('ft.add', 'idx', 'doc1', 1.0, 'fields',
-                                    'title', 'hello world',
-                                    'body', 'lorem ist ipsum'))
-
-    for _ in r.retry_with_rdb_reload():
-        prefix = 'ft'
-        env.assertExists(prefix + ':idx/hello')
-        env.assertExists(prefix + ':idx/world')
-        env.assertExists(prefix + ':idx/lorem')
+# this tests is not longer relevant
+# def testAdd(env):
+#     if env.is_cluster():
+#         raise unittest.SkipTest()
+
+#     r = env
+#     env.assertOk(r.execute_command(
+#         'ft.create', 'idx', 'schema', 'title', 'text', 'body', 'text'))
+#     env.assertTrue(r.exists('idx:idx'))
+#     env.assertOk(r.execute_command('ft.add', 'idx', 'doc1', 1.0, 'fields',
+#                                     'title', 'hello world',
+#                                     'body', 'lorem ist ipsum'))
+
+#     for _ in r.retry_with_rdb_reload():
+#         prefix = 'ft'
+#         env.assertExists(prefix + ':idx/hello')
+#         env.assertExists(prefix + ':idx/world')
+#         env.assertExists(prefix + ':idx/lorem')
 
 def testAddErrors(env):
-    env.expect('ft.create idx schema foo text bar numeric sortable').equal('OK')
+    env.expect('ft.create idx ON HASH FILTER startswith(@__key,"") schema foo text bar numeric sortable').equal('OK')
```

Comment:
Why not just use the PREFIX? also, I think that default is to index everything no?

Reply:
@rafie Can we add the default to update tests?